frontend/js/tests/utils/APIService.test.ts

```diff
@@ -709,7 +709,17 @@ describe("getFeedbackForUserForRecordings", () => {
       "new,old"
     );
     expect(window.fetch).toHaveBeenCalledWith(
-      "foobar/1/feedback/user/foo/get-feedback-for-recordings?recording_msids=bar,baz&recording_mbids=new,old"
+      "foobar/1/feedback/user/foo/get-feedback-for-recordings",
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json;charset=UTF-8",
+        },
+        body: JSON.stringify({
+          recording_msids: "bar,baz",
```

Comment:
Same comment as for backend tests. This should be array like `["bar", "baz"] instead.

Reply:
Resolved